programmers/제일-작은-수-제거하기.js

```diff
@@ -1,5 +1,5 @@
 function solution(arr) {
-  arr = arr.filter((val) => { return val != Math.min(...arr) });
+  arr = arr.filter((val) => val != Math.min(...arr));
```

Comment:
비교연산은 무조건 세개!!
두개는 최대한 지양해야합니다.
예상치못한 에러를 발생시킬 수 있습니다.
```suggestion
  arr = arr.filter((val) => val !== Math.min(...arr));
```